plugins/space-configure/src/utils/data.checklist.ts

```diff
@@ -48,8 +48,19 @@ export const createDataChecklist = async (spaceId: Flatfile.SpaceId) => {
 
   body += `\n</div>`
 
-  await api.documents.create(spaceId, {
-    title: `Data Checklist`,
-    body,
-  })
+  const { data: documents } = await api.documents.list(spaceId)
+  const checklistDocument = documents.find(
+    (document) => document.title === 'Data Checklist'
+  )
+  if (checklistDocument) {
+    await api.documents.update(spaceId, checklistDocument.id, {
+      title: `Data Checklist`,
```

Comment:
Why are we updating the title to the same thing that we checked if it was on line 53? Or is this a PUT where we have to include all the same info?

Reply:
It is a copy/paste 😆 I'll update it.